_posts/2024-01-31-practical_zephyr_dt.md

```diff
@@ -9,36 +9,36 @@ In the [previous article]({% post_url 2024-01-24-practical_zephyr_kconfig %}), w
 
 <!-- excerpt start -->
 
-In this third article of the "Practical Zephyr" series we'll see how we configure and use _hardware_. For this, Zephyr borrows another tool from the Linux kernel: [**Devicetree**](https://docs.zephyrproject.org/latest/build/dts/index.html).
+In this third article of the "Practical Zephyr" series, we'll see how we configure and use _hardware_. For this, Zephyr borrows another tool from the Linux kernel: [**Devicetree**](https://docs.zephyrproject.org/latest/build/dts/index.html).
 
-In contrast to _Kconfig_, the *Devicetree* syntax and its use are a bit more intricate. Therefore, we'll cover *Devicetree* in two articles: In this article, we'll see what a Devicetree is and how we can write our own Devicetree source files. In the next article, we'll look at so-called Devicetree _bindings_, which add semantics to our Devicetree. Be prepared for a fair bit of theory, but as usual, we'll use an example project to follow along.
```

Comment:
For my own information (no pun intended): Does it really sound weird in English to write "write our _own_ source files"? This also popped up in my Grammarly pass, but I really wanted to say that we create our "own" files that are not part of Zephyr, etc.

Reply:
In most cases it is seen as redundant, but I agree in this case it adds enough flavor to warrant staying. Will re-add.